Aptoide/aptoide-client-v8
# [MOB-73] Notification to Room

**What does this PR do?**

This PR aims at migrating the Notification table to Room. This pr can only be accepted after [MOB-260](https://github.com/Aptoide/aptoide-client-v8/pull/1293)

**Database changed?**

   Yes 

**Where should the reviewer start?**

- [x] RoomNotification.java

**How should this be manually tested?**

Open the app put rewrites for pnp onotifications and check that they open correctly

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-73](https://aptoide.atlassian.net/browse/MOB-73)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
